org.blender.Blender.json

```diff
@@ -12,7 +12,9 @@
         "--socket=x11",
         "--socket=pulseaudio",
         "--device=dri",
-        "--filesystem=home"
+        "--filesystem=home",
+        "--filesystem=/mnt",
+        "--filesystem=/media"
```

Comment:
On most GNOME systems (i.e not Debian/Ubuntu ones), external USB disks get mounted in `/run/media/$USER/$LABEL/`, so that would be missing from your patch.